platform/linux-generic/odp_stash.c

```diff
@@ -26,14 +27,16 @@
 
 ODP_STATIC_ASSERT(CONFIG_INTERNAL_STASHES < CONFIG_MAX_STASHES, "TOO_MANY_INTERNAL_STASHES");
 
-#define MAX_RING_SIZE (1024 * 1024)
 #define MIN_RING_SIZE 64
 
+#define STASH_FREE     0
+#define STASH_RESERVED 1
+#define STASH_ACTIVE   2
```

Comment:
These could be grouped into an unnamed `enum`.

Reply:
Fixed in V2.